specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -587,15 +587,14 @@
         },
         "schedule": {
           "$ref": "#/definitions/Schedule",
-          "description": "Schedule (Frequnecy, Time Window) for rule."
+          "description": "Schedule (Frequnecy, Time Window) for rule. Required for action type - AlertingAction"
         },
         "action": {
           "$ref": "#/definitions/Action",
           "description": "Action needs to be taken on rule execution."
         }
       },
       "required": [
-        "schedule",
```

Comment:
Why this property is not required anymore?

Reply:
This property is optional as the LogToMetric action of the api does not need this property.